.pylintrc

```diff
@@ -0,0 +1,15 @@
+[MESSAGES CONTROL]
+disable=R,broad-except,protected-access,unused-argument,redefined-builtin,missing-docstring,invalid-name,ungrouped-imports,wrong-import-order,bad-continuation,global-statement
```

Comment:
You can safely remove: `redefined-builtin`, `unused-argument` from here.
(I think it's better to opt-in for it, if they're ever needed)
`ungrouped-imports` - can be removed as well (it doesn't conflict with `isort`)

Regarding `broad-except` - I think the same, it's up to you (currently it does appear in several places).

Reply:
There are a bunch of unused-argument  (see flask_app/errors.py:7, for example)

Reply:
`broad-except` can be done in a separate MR, it's more complicated